docs/pages/cloud/introduction.mdx

```diff
@@ -1,16 +1,59 @@
 ---
-title: Cloud
-description: Teleport as a service
-h1: Teleport cloud
+title: Teleport Cloud
+description: Teleport Cloud is a fully managed deployment of Teleport.
```

Comment:
To be more precise: is a deployment of Teleport control plane managed by Teleport Cloud team